modules/server_processes_manager/php/server_processes_manager.class.inc

```diff
@@ -114,7 +96,10 @@ class Server_Processes_Manager extends \NDB_Menu_Filter
         $deps    = parent::getJSDependencies();
         return array_merge(
             $deps,
-            array($baseURL . "/server_processes_manager/js/columnFormatter.js")
+            [
```

Comment:
we elect to use `array()` everywhere. Please replace the `[]`

Reply:
:(